src/storage/src/hummock/local_version/local_version_manager.rs

```diff
@@ -115,6 +120,7 @@ impl LocalVersionManager {
         sstable_store: SstableStoreRef,
         hummock_meta_client: Arc<dyn HummockMetaClient>,
         event_sender: UnboundedSender<HummockEvent>,
+        compaction_group_client: Arc<CompactionGroupClientImpl>,
```

Comment:
Do we still need a `compaction_group_client` in the CN? Seems we can get the table id -> group id mapping from the hummock version. 

Since we have supported putting a sstable into multiple compaction groups, the  table id -> group id mapping does not have to be accurate. If a table id to write is not found, we can assign a default compaction group in local compaction.